cargo/src/bin/cargo_audit.rs

```diff
@@ -24,69 +24,6 @@
 #![deny(clippy::manual_let_else)]
 #![allow(clippy::unreachable)]
 
-use std::io;
-use std::io::IsTerminal;
-
-use clap::Parser;
-use terminfo::{capability as cap, Database};
-#[allow(unused_imports)]
-use tracing::{debug, error, info, warn, Level};
-use tracing_subscriber::EnvFilter;
-
-use obs_service_cargo::audit::Audit;
-use obs_service_cargo::audit::AuditOpts;
-
-fn main() -> io::Result<()> {
-    let args = AuditOpts::parse();
-
-    let terminfodb = Database::from_env().map_err(|e| {
-        error!(err = ?e, "Unable to access terminfo db. This is a bug!");
-        io::Error::new(
-            io::ErrorKind::Other,
-            "Unable to access terminfo db. This is a bug!",
-        )
-    })?;
-
-    let is_termcolorsupported = terminfodb.get::<cap::MaxColors>().is_some();
-    let to_color = matches!(std::io::stdout().is_terminal(), true if {
-        let coloroption = &args.color;
-        match coloroption {
-            clap::ColorChoice::Auto => is_termcolorsupported,
-            clap::ColorChoice::Always => true,
-            clap::ColorChoice::Never => false,
-        }
-    });
-
-    let filter_layer = EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info"));
-
-    let builder = tracing_subscriber::fmt()
-        .with_level(true)
-        .with_ansi(to_color)
-        .with_env_filter(filter_layer)
-        .with_level(true);
-
-    let builder = if cfg!(debug_assertions) {
-        builder.with_file(true).with_line_number(true)
-    } else {
-        builder
-    };
-
-    builder.init();
-
-    info!("🕵️ Starting OBS Service Cargo Audit.");
-    debug!(?args);
-
-    let service_file = std::env::current_dir()
-        .map_err(|err| {
-            error!(?err, "Unable to determine current working directory");
-            err
-        })?
-        .join("_service");
-    let opts = args.clone().generate_opts(&service_file)?;
-
-    for opt in opts {
-        opt.src.run_audit(&args)?;
-    }
-
-    Ok(())
+fn main() {
+    // Do nothing, really well.
```

Comment:
We should probably add a warning that people have to remove 

```xml
<service name="cargo_audit" />
```

But they won't notice anyway since it dead 🥴

Reply:
I was thinking we wait for a bit before we warn because there could be some people on the python versions for some time still. So we add the warning in a few months ?